webanno/webanno
# #1736 Optimize ProjectServiceImpl.listProjectUsersWithPermissions

The method queries the db for all usernames with a certain permission and then goes through all names and asks the db first if the user exists and then gets it.

**What's in the PR**
* Using one join instead of retrieving users with additional db queries

**How to test manually**
* Check that feature which use the two `listProjectUsersWithPermissions` methods still work

**Automatic testing**
